### ISSUE TITLE
Password for restored accounts 

### ISSUE THREAD

**Issue by KeeJef** [loki-project/loki-messenger#220]
Currently there is no way to add a password for an account that is created from a Seed phrase, this should be added to the new registration UI, this will depend on another issue with a proposal to overhaul the registration UI that will be created later so lets wait on this one

**Comment by BeaudanBrown** [comment]
You can always set a password through the main screen (accordion menu from clicking on your pubkey or nickname), but possibly we should just include this from the restore seed screen as well?
But it would need to be made very clear that this is setting a NEW password for this local instance of the app, not requesting a password for the seed you are trying to restore
